0-ad-history-mod-package/simulation/templates/mixins/encyclopedia/encyclopedia_siege_tower.xml

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Entity>
+  <Identity>
+    <History>
+        Because siege towers were so bulky, they were often constructed on site from materials gathered from the surrounding woods. Although they came in different sizes, they were as tall as or taller than the besieged city`s walls.
```

Comment:
```suggestion
        Because siege towers were so bulky, they were often constructed on site from materials gathered from the surrounding area. Although they came in different sizes, they were always at least as tall as the besieged city`s walls.
```